tests/integration/testContainerConfig.ts

```diff
@@ -0,0 +1,62 @@
+// import { container } from 'tsyringe';
```

Comment:
why the remarks?

Reply:
should have been deleted , i removed it as it the old way to register services in tests